lib/directory.coffee

```diff
@@ -266,7 +266,10 @@ class Directory
   squashDirectoryNames: (fullPath) ->
     squashedDirs = [@name]
     loop
-      contents = fs.listSync fullPath
+      try
+        contents = fs.listSync fullPath
+      catch error
+        break
```

Comment:
We should probably log the error as a warning instead of throwing it, similar to what's done when trying to save files that we don't have permissions for.